ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -0,0 +1,199 @@
+/**
+ * A screen where user after login (with CIE) can set email address if is not present in the profile.
+ */
+
+import { none, Option, some } from "fp-ts/lib/Option";
+import * as pot from "italia-ts-commons/lib/pot";
+
+import { Content, Form, Text, View } from "native-base";
+
+import * as React from "react";
+
+import {
+  Alert,
+  KeyboardAvoidingView,
+  Platform,
+  StyleSheet
+} from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import { LabelledItem } from "../../components/LabelledItem";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import H4 from "../../components/ui/H4";
+import Markdown from "../../components/ui/Markdown";
+import I18n from "../../i18n";
+import { abortOnboarding, emailInsert } from "../../store/actions/onboarding";
+import { ReduxProps } from "../../store/actions/types";
+import { profileSelector } from "../../store/reducers/profile";
+import { GlobalState } from "../../store/reducers/types";
+import customVariables from "../../theme/variables";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+
+type Props = ReduxProps & OwnProps & ReturnType<typeof mapStateToProps>;
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1
+  },
+  content: {
+    flex: 1
+  },
+  horizontalPadding: {
+    paddingHorizontal: customVariables.contentPadding
+  },
+  boldH4: {
+    fontWeight: customVariables.textBoldWeight,
+    paddingTop: customVariables.spacerLargeHeight
+  }
+});
+
+const EMPTY_EMAIL = "";
+
+const INITIAL_STATE: State = {
+  email: none
+};
+
+type State = Readonly<{
+  email: Option<string>;
+}>;
+
+/**
+ * A valid Email
+ */
+export const Email = RegExp(
```

Comment:
Use [EmailString](https://github.com/teamdigitale/io-ts-commons/blob/22b84b1bd07111d30da9705bef850b5c97bf12c5/src/strings.ts#L75) could be a better solution

Reply:
changed in 70fe1d0a